packages/core/src/__integrationtests__/Did.spec.ts

```diff
@@ -336,34 +340,44 @@ it('creates and updates DID, and then reclaims the deposit back', async () => {
 describe('DID migration', () => {
   it('migrates light DID with ed25519 auth key and encryption key', async () => {
     const { sign, authenticationKey } = makeSigningKeyTool('ed25519')
-    const lightDidDetails = LightDidDetails.fromDetails({
+    const lightDidDetails = fromDetails({
       authenticationKey,
       encryptionKey: makeEncryptionKeyTool(
         '0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb'
       ).keypair,
     })
 
-    const migratedFullDid = await lightDidDetails.migrate(
+    const creationTx = await DidChain.generateCreateTxFromCreationDetails(
```

Comment:
think I found it, I assume we would recommend using `generateCreateTxFromDidDetails` after creating a light did

Reply:
Good catch! This change was done while removing the `migrate()`, and I just inlined whatever it was doing. It was not using the `generateCreateTxFromDidDetails` however, because it wanted to control whether the encryption key and service endpoints should be migrated. We’re not really using this control, so `generateCreateTxFromDidDetails` is really a better candidate here!

Reply:
I’ve pushed an improvement for that

Reply:
nice one!